bundles/ui/org.eclipse.smarthome.ui/OSGI-INF/chartprovider.xml

```diff
@@ -20,4 +20,5 @@
    <service>
       <provide interface="org.eclipse.smarthome.ui.chart.ChartProvider"/>
    </service>
+   <reference bind="setTimeZoneProvider" cardinality="1..1" interface="org.eclipse.smarthome.core.i18n.TimeZoneProvider" name="TimeZoneProvider" policy="static" unbind="unsetTimeZoneProvider"/>
```

Comment:
I would like to kindly ask you if you can also convert this to DS-annotations but the `OSGI-INF` folder contains 4 files, so I leave it to you if you want to convert all of them or at least just this one and ignore this single created file from the Eclipse IDE in the `.gitignore` file.